packages/angular-material/src/controls/boolean.renderer.ts

```diff
@@ -34,7 +34,6 @@ import { isBooleanControl, JsonFormsState, RankedTester, rankWith } from '@jsonf
         <mat-checkbox
             (change)="onChange($event)"
             [checked]="isChecked()"
-            [disabled]="!enabled"
```

Comment:
I think we can't remove the disable here as the checkbox is not controlled by the reactive form control `form`